tests/system/action/user/test_save_saml_account.py

```diff
@@ -526,13 +523,13 @@ def test_update_user_with_membership(self) -> None:
 
     def test_create_user_invalid_meeting(self) -> None:
         """silent fail, user created and logged in"""
-        organization = self.get_model("organization/1")
-        for key in ("meta_deleted", "meta_position"):
-            organization.pop(key, None)
-        organization["saml_attr_mapping"]["meeting"]["external_id"] = "Kreistag"
-        self.set_models({"organization/1": organization})
+        self.organization["saml_attr_mapping"]["meeting"]["external_id"] = "Kreistag"  # type: ignore
```

Comment:
Why do you need the `type: ignore` here? I would prefer this to be solved by assigning the correct type in the constructor (`dict[str, Any]` should suffice)